yarn-project/end-to-end/src/e2e_account_contracts.test.ts

```diff
@@ -56,7 +55,8 @@ function itShouldBehaveLikeAnAccountContract(
       const { logger, pxe } = context;
       logger('Calling public function...');
       await child.methods.pubIncValue(42).send().wait({ interval: 0.1 });
-      expect(toBigInt((await pxe.getPublicStorageAt(child.address, new Fr(1)))!)).toEqual(42n);
+      const storedValue = await pxe.getPublicStorageAt(child.address, new Fr(1));
```

Comment:
Heres another area where there is inconsistency:

getPublicStorage at sometimes has a bigint and sometimes has a field, as this is just an index, im in favour of this being just a bigint

![image](https://github.com/AztecProtocol/aztec-packages/assets/47148561/383c1534-afcb-42e0-bd44-d39518468682)